cmd/gcp-controller-manager/app/gke_approver.go

```diff
@@ -475,12 +475,15 @@ func validateTPMAttestation(opts approverOptions, csr *capi.CertificateSigningRe
 	}
 
 	// TODO(awly): get AIK public key from GCE API.
-	// TODO(awly): verify aikCert to root CA.
```

Comment:
It looks like the code you added does verify the AIK cert; did you mean to put a TODO for the EK cert in here?

Reply:
Not sure what you mean, i removed a TODO that's now done

Reply:
Yeah, I think I misread the diff.

Part of the question still stands, though: do you need to verify the EK cert?

Reply:
We only pass the AIK cert